tools/add_triage_signature.py

```diff
@@ -683,6 +662,10 @@ def get_issues(jclient, issue, only_recent):
     return get_all_triage_tickets(jclient, only_recent=only_recent)
 
 
+def browse(issue_key: str):
```

Comment:
we need a more indicative name than ``browse``

Reply:
sure